terms.md

```diff
@@ -65,28 +65,19 @@ permalink: /terms/
         ADMINISTERED BY THE U.S. GENERAL SERVICES ADMINISTRATION
       </p>
       <p>
-      Version 1.0, updated: 03/11/2022
+      Version 1.1, updated: 08/12/2024
       </p>
       <h3 id="summary">
-        1.0 SERVICE SUMMARY
+        1.1 SERVICE SUMMARY
       </h3>
       <p>
-        “Touchpoints” is an open-source software application created and maintained by the U.S.
-        General Services Administration (GSA) provided to agencies at no-cost for the terms of service
-        outlined in this document. Touchpoints enables government agencies to solicit and process
-        user feedback to support the continuous improvement of public systems, services, processes,
-        and policies.
+        “Touchpoints” is an open-source software application created and maintained by the U.S. General Services Administration (GSA) provided to agencies at no-cost for the terms of service outlined in this document. Touchpoints enables government agencies to solicit and process user feedback to support the continuous improvement of public systems, services, processes, and policies.
       </p>
       <p>
-        The following terms of service (“Terms”) governing GSA’s Touchpoints website and services,
-        including the content, documentation, code, and related materials are offered subject to your
-        acceptance of the Terms, as well as any relevant sections of the Touchpoints Site Policies
-        (collectively, the “Agreement”). Access to or use of Touchpoints services or its content
-        constitutes acceptance to this Agreement.
+        The following terms of service (“Terms”) governing GSA’s Touchpoints website and services, including the content, documentation, code, and related materials are offered subject to your acceptance of the Terms, as well as any relevant sections of the Touchpoints Site Policies (collectively, the “Agreement”). Access to or use of Touchpoints services or its content constitutes acceptance to this Agreement.
```

Comment:
retaining the line breaks minimizes the diffs here

sometimes code editors wrap the text